charts/monitoring/prometheus/config/scraping/pods.yaml

```diff
@@ -21,6 +21,13 @@ relabel_configs:
   regex: '{{ .Release.Namespace }};node-exporter'
   action: drop
 
+# drop minio, if it's running with TLS
+{{ if .Values.prometheus.scraping.minio.tls.enabled }}
+- source_labels: [__meta_kubernetes_namespace, __meta_kubernetes_pod_label_app]
+  regex: '{{ .Values.prometheus.scraping.minio.namespace | default "minio" }};{{ .Values.prometheus.scraping.minio.appLabel | default "minio" }}'
```

Comment:
What do you think about just setting the defaults in `values.yaml` instead of using `| default`?

Reply:
Both should work.